desc/protoprint/print.go

```diff
@@ -1696,9 +1709,14 @@ func (p *Printer) printOption(name string, optVal interface{}, w *writer, indent
 			return
 		}
 
-		// multi-line form
-		m.Compact = false
-		str = m.Text(optVal)
+		//multi-line form
+		m.Multiline = true
```

Comment:
Comment is no longer necessary due to new name of the field
```suggestion
		m.Multiline = true
```